calico/maintenance/monitor/monitor-component-metrics.md

```diff
@@ -31,7 +31,26 @@ You can configure Felix, Typha, and/or kube-controllers to provide metrics to Pr
 
 ### Before you begin...
 
-In this tutorial we assume that you have completed all other introductory tutorials and possess a running Kubernetes cluster with {{site.prodname}}, calicoctl and kubectl installed.
+In this tutorial we assume that you have completed all other introductory tutorials and possess a running Kubernetes cluster with {{site.prodname}}. You can either use `kubectl` or `calicoctl` to perform the following steps. Depending on which tool you would like to use, make sure you have the necessary prerequisites as shown below.
+
+
+{% tabs %}
+<label:kubectl,active:true>
+<%
+If you wish to modify {{site.prodname}} configurations with `kubectl` binary you need to make sure you have the {{site.prodname}} API server in your cluster. API Server allows you to manage {{site.prodname}} spesific resources that exists within the `projectcalico.org/v3` api group.
```

Comment:
"API Server" -> "The API server"

"spesific" -> "specific"

Reply:
Actually can just change to "allows you to manage resources within the `projectcalico.org/v3` api group"